cluster/src/main/scala/io/snappydata/gemxd/SparkSQLExecuteImpl.scala

```diff
@@ -483,7 +483,7 @@ object SnappyContextPerConnection {
     val context = connectionIdMap.get(connectionID)
     if (context != null) context
     else {
-      val snc = SnappyContext.getOrCreate(null: SparkContext).newSession()
+      val snc = SnappyContext(null: SparkContext).newSession()
```

Comment:
@hbhanawat I don't think separate JDBC connections should be sharing the same UI tab in any case which is currently wrong behaviour. Only SQL from the same connection should share which was one of the purpose of connectionIdMap (to have same SnappyContext for same JDBC connection). So we should just remove newSession() here in my opinion.